src/domain/core/src/services/dependency_graph_repository.rs

```diff
@@ -23,6 +23,6 @@ pub trait DependencyGraphRepository: Sync + Send {
 /////////////////////////////////////////////////////////////////////////////////////////
 
 pub type DatasetDependenciesIDStream<'a> =
-    Pin<Box<dyn Stream<Item = Result<(DatasetID, DatasetID), InternalError>> + Send + 'a>>;
+    Pin<Box<dyn Stream<Item = Result<(DatasetID, Vec<DatasetID>), InternalError>> + Send + 'a>>;
```

Comment:
Nit pic: I prefer an explicit struct with good field names instead of a tuple with a growing compexity

Reply:
Fixed via [f5773bc](https://github.com/kamu-data/kamu-cli/pull/421/commits/f5773bc3e1b7cb68b13d62cfe4bafdf551d5b390)